tests/scripts/utils.sh

```diff
@@ -0,0 +1,23 @@
+# Wait for the ols api server to respond 200 on the readiness endpoint
+# $1 - url of the ols server to poll
+function wait_for_ols() {
+    # Don't exit on error while polling the OLS server
+    # Curl will return error exit codes until OLS is available
+    set +e
+    STARTED=0
+    for i in {1..20}; do
+    echo Checking OLS readiness, attempt "$i" of 20
+    curl -sk --fail "$1/readiness"
```

Comment:
`if curl -sk --fail "$1/readiness";` perhaps?

Reply:
the current form feels more easily read/understood to me